config/deploy.rb

```diff
@@ -15,6 +15,8 @@
 
 set :branch, 'master'
 
+set :passenger_restart_with_touch, true
```

Comment:
You can also remove the `deploy:restart` task below; capistrano-passenger will magically do that for you.